_Group Projects/Group_2/Week_8/simulation.do

```diff
@@ -0,0 +1,73 @@
+clear all
+
+*******************************************************************************
+*************************  Change directory  **********************************
+*******************************************************************************
+
+cd "/Users/felipe.germanos/Documents/Felipe/Georgetown/Semesters/7thSemester/Research Design/Week 8"
+
+*******************************************************************************
+*************** Setting up random seed and running program  *******************
+*******************************************************************************
+
+
+set seed 15771 // generated using random.org
+do "program.do"
+
+*******************************************************************************
+**** Running the program several times and storing results in a csv files  ****
+*******************************************************************************
+
+
+/* First, we obtain means for 10, 50, 100, and 500 runs of the program to determine whether there is error convergence; error should decrease for larger numbers of runs */
+foreach j in 10 50 100 500 {
+	forval i = 1/`j' {
+	clear
+	quietly pmt `j'
+	// get the inclusion and exclusion errors 
+	mat results = nullmat(results) ///
+		\[`i',`r(ie)', `r(ee)']	
+		
+}
```

Comment:
Why would the errors converge for more runs here? This should be explained in the `README` as well as your findings. You might consider investigating whether the _inclusion error_ is changing, or whether the _standard errors_ around your calculation of the inclusion error are changing.